editor/sidebar/post-settings/index.js

```diff
@@ -25,10 +25,14 @@ const PostSettings = ( { toggleSidebar } ) => {
 		<Panel>
 			<PanelHeader label={ __( 'Post Settings' ) } >
 				<div className="editor-sidebar-post-settings__icons">
-					<IconButton icon="admin-settings" />
+					<IconButton
+						icon="admin-settings"
+						label={ __( 'WordPress settings' ) }
+					/>
 					<IconButton
 						onClick={ toggleSidebar }
 						icon="no-alt"
+						label={ __( 'Close post settings sidebar' ) }
```

Comment:
Sorry I forgot to reply... well I thought to give some feedback about the "what" post settings is... the wording can be certainly reviewed and improved, I have no strong opinions about that. We should also consider screen readers are used also by low vision users.